07LessonSeven/in-class-project/css/my-style.css

```diff
@@ -0,0 +1,63 @@
+  .nasa-background {
+      background-image: url("../img/nasa-background.jpg");
+      height: 450px;
+      width: 850px;
+      float: top;
```

Comment:
"top" is not a value for float. if you float it left, the page looks better (but you'll also have to float the main or section or clear this one for the margin to take place).